src/OrchardCore.Modules/OrchardCore.Media/Models/MediaCommands.cs

```diff
@@ -0,0 +1,92 @@
+#nullable enable
+
+namespace OrchardCore.Media.Models;
+
+public sealed class MediaCommands
```

Comment:
I think we should keep this `internal` if possible. It's just an implementation detail.